internal/providers/github/oauth/oauth.go

```diff
@@ -34,8 +34,12 @@ import (
 	provifv1 "github.com/stacklok/minder/pkg/providers/v1"
 )
 
-// Github is the string that represents the GitHubOAuth provider
-const Github = "github"
+const (
```

Comment:
We do have a `ProviderClass` enum in `minder.proto`. Could we reuse that, instead of re-defining the options here?